cli/cmd/testdata/inject_emojivoto_deployment_tls.golden.yml

```diff
@@ -0,0 +1,120 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  creationTimestamp: null
+  name: web
+  namespace: emojivoto
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: web-svc
+  strategy: {}
+  template:
+    metadata:
+      annotations:
+        conduit.io/created-by: conduit/cli undefined
+        conduit.io/proxy-version: testinjectversion
+      creationTimestamp: null
+      labels:
+        app: web-svc
+        conduit.io/control-plane-ns: conduit
+        conduit.io/proxy-deployment: web
+    spec:
+      containers:
+      - env:
+        - name: WEB_PORT
+          value: "80"
+        - name: EMOJISVC_HOST
+          value: emoji-svc.emojivoto:8080
+        - name: VOTINGSVC_HOST
+          value: voting-svc.emojivoto:8080
+        - name: INDEX_BUNDLE
+          value: dist/index_bundle.js
+        image: buoyantio/emojivoto-web:v3
+        name: web-svc
+        ports:
+        - containerPort: 80
+          name: http
+        resources: {}
+      - env:
+        - name: CONDUIT_PROXY_LOG
+          value: warn,conduit_proxy=info
+        - name: CONDUIT_PROXY_BIND_TIMEOUT
+          value: 10s
+        - name: CONDUIT_PROXY_CONTROL_URL
+          value: tcp://proxy-api.conduit.svc.cluster.local:8086
+        - name: CONDUIT_PROXY_CONTROL_LISTENER
+          value: tcp://0.0.0.0:4190
+        - name: CONDUIT_PROXY_METRICS_LISTENER
+          value: tcp://0.0.0.0:4191
+        - name: CONDUIT_PROXY_PRIVATE_LISTENER
+          value: tcp://127.0.0.1:4140
+        - name: CONDUIT_PROXY_PUBLIC_LISTENER
+          value: tcp://0.0.0.0:4143
+        - name: CONDUIT_PROXY_POD_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        - name: CONDUIT_PROXY_TLS_TRUST_ANCHORS
+          value: /var/conduit-io/trust-anchors/trust-anchors.pem
+        - name: CONDUIT_PROXY_TLS_CERT
+          value: /var/conduit-io/identity/certificate.crt
+        - name: CONDUIT_PROXY_TLS_PRIVATE_KEY
+          value: /var/conduit-io/identity/private-key.p8
+        - name: CONDUIT_PROXY_TLS_POD_IDENTITY
+          value: web.deployment.$CONDUIT_PROXY_POD_NAMESPACE.conduit-managed.conduit.svc.cluster.local
+        - name: CONDUIT_PROXY_CONTROLLER_NAMESPACE
+          value: conduit
+        - name: CONDUIT_PROXY_TLS_CONTROLLER_IDENTITY
+          value: controller.deployment.conduit.conduit-managed.conduit.svc.cluster.local
```

Comment:
@briansmith Can you verify that all of these env vars now look correct?

Reply:
> @briansmith Can you verify that all of these env vars now look correct?

Yes, they look like they are expected to now.